.github/workflows/run-tests-on-pr.yaml

```diff
@@ -0,0 +1,30 @@
+name: Run tests for pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  pr_test:
+    uses: ./.github/workflows/run-on-testing-farm.yaml
+    with:
+      distro: "fedora-37"
```

Comment:
Should be `Fedora-37`, it is case sensitive.

Reply:
Done